db/changes.go

```diff
@@ -386,6 +386,8 @@ func (db *DatabaseCollectionWithUser) changesFeed(ctx context.Context, singleCha
 
 	feed := make(chan *ChangeEntry, 1)
 
+	options.LoggingCtx = ctx
```

Comment:
Is this doen so we don't have to change the `GetChanges` API?

Reply:
Yep, pretty much.

This context was already being set from the BLIP codepath, just not the REST API changes codepath.